sequencer/Cargo.toml

```diff
@@ -73,3 +73,4 @@ url = "2.3"
 
 # Dependencies for feature `testing`
 hotshot-testing = { workspace = true, optional = true }
+sha3 = "0.10.8"
```

Comment:
Try to keep dependencies grouped together by feature flag. This looks like a required dependency but it's listed under the comment "Dependencies for feature `testing`". Should be moved up to the big group of dependencies above.